src/main/java/uk/gov/pay/adminusers/service/InviteService.java

```diff
@@ -93,6 +93,18 @@ public void reGenerateOtp(ResendOtpRequest resendOtpRequest) {
             throw notFoundInviteException(resendOtpRequest.getCode());
         }
     }
+    
+    @Transactional
+    public void sendOtp(InviteEntity invite) {
+        OtpNotifySmsTemplateId templateId = (invite.getService().isPresent()) ? CREATE_USER_IN_RESPONSE_TO_INVITATION_TO_SERVICE : SELF_INITIATED_CREATE_NEW_USER_AND_SERVICE;
+
+        try {
+            String notificationId = notificationService.sendSecondFactorPasscodeSms(invite.getTelephoneNumber(), invite.getPassword(), templateId);
```

Comment:
This doesn't look right, we shouldn't be sending the password in the SMS message! We need to generate an otp code from the `otp_key` on the invite. Take a look at the existing code to see how we do this.

Reply:
Yeah, I did think this was weird.  I think I did look at existing code but misinterpreted it.  I've now sorted it out.